From 0613ae007915d4669a44edb133c7a417dee8c71d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Sun, 24 Feb 2013 12:54:05 +0200
Subject: ARM: i.MX6: cm-fx6: fix I2C1 pinmux

cm-fx6 uses EIM_D21 for I2C1_SCL and EIM_D28 for I2C1_SDA.
Configure the right pins.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/board-cm-fx6-dl.h |    6 ++----
 arch/arm/mach-mx6/board-cm-fx6-q.h  |    6 ++----
 2 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-mx6/board-cm-fx6-dl.h b/arch/arm/mach-mx6/board-cm-fx6-dl.h
index f1536ad..287f5eb 100644
--- a/arch/arm/mach-mx6/board-cm-fx6-dl.h
+++ b/arch/arm/mach-mx6/board-cm-fx6-dl.h
@@ -87,8 +87,8 @@ static iomux_v3_cfg_t mx6dl_arm2_pads[] = {
 	MX6DL_PAD_EIM_D19__GPIO_3_19,	/*SS1*/
 
 	/* I2C1 */
-	MX6DL_PAD_CSI0_DAT8__I2C1_SDA,
-	MX6DL_PAD_CSI0_DAT9__I2C1_SCL,
+	MX6DL_PAD_EIM_D21__I2C1_SCL,
+	MX6DL_PAD_EIM_D28__I2C1_SDA,
 
 	/* I2C2 */
 	MX6DL_PAD_KEY_COL3__I2C2_SCL,
@@ -129,8 +129,6 @@ static iomux_v3_cfg_t mx6dl_arm2_pads[] = {
 	/* UART2 */
 	MX6DL_PAD_EIM_D26__UART2_RXD,
 	MX6DL_PAD_EIM_D27__UART2_TXD,
-	MX6DL_PAD_EIM_D28__UART2_RTS,
-	MX6DL_PAD_EIM_D29__UART2_CTS,
 
 	/* PWM1 */
 	MX6DL_PAD_GPIO_9__PWM1_PWMO,
diff --git a/arch/arm/mach-mx6/board-cm-fx6-q.h b/arch/arm/mach-mx6/board-cm-fx6-q.h
index 8e86a34..f436918 100644
--- a/arch/arm/mach-mx6/board-cm-fx6-q.h
+++ b/arch/arm/mach-mx6/board-cm-fx6-q.h
@@ -87,8 +87,8 @@ static iomux_v3_cfg_t mx6q_arm2_pads[] = {
 	MX6Q_PAD_EIM_D19__GPIO_3_19,	/*SS1*/
 
 	/* I2C1 */
-	MX6Q_PAD_CSI0_DAT8__I2C1_SDA,
-	MX6Q_PAD_CSI0_DAT9__I2C1_SCL,
+	MX6Q_PAD_EIM_D21__I2C1_SCL,
+	MX6Q_PAD_EIM_D28__I2C1_SDA,
 
 	/* I2C2 */
 	MX6Q_PAD_KEY_COL3__I2C2_SCL,
@@ -129,8 +129,6 @@ static iomux_v3_cfg_t mx6q_arm2_pads[] = {
 	/* UART2 */
 	MX6Q_PAD_EIM_D26__UART2_RXD,
 	MX6Q_PAD_EIM_D27__UART2_TXD,
-	MX6Q_PAD_EIM_D28__UART2_RTS,
-	MX6Q_PAD_EIM_D29__UART2_CTS,
 
 	/* PWM1 */
 	MX6Q_PAD_GPIO_9__PWM1_PWMO,
-- 
1.7.9.5

